core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/stores/dot-experiments-store.service.ts

```diff
@@ -1,21 +1,175 @@
+    }));
+    readonly archiveExperimentById = this.updater((state, experimentId: string) => ({
+        ...state,
+        experiments: state.experiments.map((exp) =>
+            experimentId === exp.id ? { ...exp, status: DotExperimentStatusList.ARCHIVED } : exp
+        )
+    }));
+
+    // Effects
+    readonly loadExperiments = this.effect((trigger$) =>
+        trigger$.pipe(
+            withLatestFrom(this.getSelectedPageId$),
+            switchMap(([, pageId]) => {
+                return this.dotExperimentsService.get(pageId).pipe(
+                    tap(() => this.setComponentStatus(LoadingState.LOADING)),
+                    tapResponse(
+                        (experiments) => this.setExperiments(experiments),
+                        (error: HttpErrorResponse) => throwError(error),
+                        () => this.setComponentStatus(LoadingState.LOADED)
+                    )
+                );
+            })
+        )
+    );
+    readonly deleteExperiment = this.effect((experimentId$: Observable<string>) => {
+        return experimentId$.pipe(
+            tap(() => this.setComponentStatus(LoadingState.LOADING)),
+            switchMap((id) =>
+                this.dotExperimentsService.delete(id).pipe(
+                    tapResponse(
+                        () => this.deleteExperimentById(id),
+                        (error) => throwError(error),
+                        () => this.setComponentStatus(LoadingState.LOADED)
+                    ),
+                    catchError(() => EMPTY)
+                )
+            )
+        );
+    });
+    readonly archiveExperiment = this.effect((experimentId$: Observable<string>) => {
+        return experimentId$.pipe(
+            tap(() => this.setComponentStatus(LoadingState.LOADING)),
+            switchMap((id) =>
+                this.dotExperimentsService.archive(id).pipe(
+                    tapResponse(
+                        () => this.archiveExperimentById(id),
+                        (error) => throwError(error),
+                        () => this.setComponentStatus(LoadingState.LOADED)
+                    ),
+                    catchError(() => EMPTY)
+                )
+            )
+        );
+    });
+
+    // VM's
+    readonly vmShell$ = this.select(
+        this.getStatus$,
+        this.getSelectedPageTitle$,
+        (status, currentTitlePage) => ({
+            status,
+            currentTitlePage
+        })
+    );
+
+    readonly vmExperimentList$: Observable<VmListExperiments> = this.select(
+        this.isLoading$,
+        this.getExperiments$,
+        this.getExperimentsFilteredAndGroupedByStatus$,
+        this.getExperimentsFilterStatusList$,
+        (isLoading, experiments, experimentsFiltered, experimentsFilterStatus) => ({
+            isLoading,
+            experiments,
+            experimentsFiltered,
+            experimentsFilterStatus
+        })
+    );
```

Comment:
Looks like you need a store for the shell and another one or for the list, stores should not be shared between components.